src/clients/cassandra/CassandraConnector.js

```diff
@@ -1,19 +1,60 @@
 'use strict';
 
 const Promise = require('promise');
+const cassandra = require('cassandra-driver');
+const asyncEachLimit = require('async/eachLimit');
+const _ = require('lodash/array');
```

Comment:
you should only be importing the lodash function that you're using, as lodash is a massive library.